GVFS/GVFS.Upgrader/UpgradeOrchestrator.cs

```diff
@@ -90,7 +90,6 @@ public void Execute()
                     {
                         mountError = Environment.NewLine + "WARNING: " + mountError;
                         this.output.WriteLine(mountError);
-                        this.ExitCode = ReturnCode.Success;

```

Comment:
Does this change also need to go into the other auto-upgrade feature branch?

Reply:
Yes... It is not related to name change I know, I just happened to discover it while testing this PR.